envs/snv.yaml

```diff
@@ -3,4 +3,4 @@ channels:
   - defaults
   - conda-forge
 dependencies:
-  - shorah
+  - shorah =1.99.0
```

Comment:
Why do we need this? specially, for software that is maintained in the group we can omit the versions, no?